Homework2/Homework2/ConsoleApp.cs

```diff
@@ -0,0 +1,222 @@
+                                            break;
+                                        default:
+                                            consoleRobot.WriteMassage("- Your answer should be 1 to 3, try again");
+                                            break;
+                                    }
+                                    
+                                    managedProject.ShowProject();
+                                }
+
+                                break;
+                            case "2":
+                                consoleRobot.WriteMassage("Enter a new value and sequence number");
+                                
+                                managedTasks.ReplaceDescription(consoleRobot.GetNewValue(), consoleRobot.GetSequenceNumber());
+                                managedTasks.ShowTasks();
+
+                                break;
+                            case "3":
+                                while (true)
+                                {
+                                    consoleRobot.WriteMassage("Choose which options you need to replace ((1 - name, 2 - post, 3 - stop choosing)");
+                                    string? answerWhatReplaceInStaff = consoleRobot.ReadInput();
+
+                                    if (answerWhatReplaceInStaff == "3")
+                                        break;
+
+                                    switch (answerWhatReplaceInStaff)
+                                    {
+                                        case "1":
+                                            managedStaff.ReplaceName(consoleRobot.GetNewValue(), consoleRobot.GetSequenceNumber());
+                                            break;
+                                        case "2":
+                                            managedStaff.ReplacePost(consoleRobot.GetNewValue(), consoleRobot.GetSequenceNumber());
+                                            break;
+                                        default:
+                                            consoleRobot.WriteMassage("- Your answer should be 1 to 3, try again");
+                                            break;
+                                    }
+                                    
+                                    managedStaff.ShowStaff();
+                                }
+
+                                break;
+                            default:
+                                consoleRobot.WriteMassage("- Your answer should be 1 to 3, try again");
+                                break;
+                        }
+                        
+                        break;
+                    case "5":
+                        return;
+                    default:
+                        consoleRobot.WriteMassage("- Your answer should be 1 to 6,  try again");
+                        break;
+                }
+            }
+        }
```

Comment:
It's too much for a single method, it deserves to be at least a class with a bunch of methods, or maybe multiple classes.
I would start with extracting switch-cases into separate methods depending on their responsibilities.

Reply:
Fixed